apps/expo/src/app/index.tsx

```diff
@@ -6,6 +6,9 @@ import { Stack } from "expo-router";
 
 import registerForPushNotificationsAsync from "~/notifications/registerNotifications";
 
+// const imageUrl = 'https://static.wikia.nocookie.net/acc-official-database/images/9/91/El_gato.jpg/revision/latest?cb=20220709001857';
+// const imageSourceProp: ImageSourcePropType = { uri: imageUrl };
```

Comment:
Let's not leave in commented code for better style. If you need to save these you can put them in the ticket description

Reply:
just deleted!